flocker/testtools/ssh.py

```diff
@@ -29,6 +29,26 @@
 if_conch = skipIf(not _have_conch, "twisted.conch must be useable.")
 
 
+@if_conch
+def generate_ssh_key(key_file):
+    """
+    Generate a ssh key.
+
+    :param FilePath key_file: Path to create ssh key at.
+
+    :return Key: The generated key.
+    """
+    check_call(
+        [b"ssh-keygen",
+         # Specify the path where the generated key is written.
+         b"-f", key_file.path,
+         # Specify an empty passphrase.
+         b"-N", b"",
+         # Generate as little output as possible.
```

Comment:
That is, it is certainly possible, but I think replacing `check_call` with something that logs is more work than is warranted here.